test/SharedFx.UnitTests/SharedFx.UnitTests.csproj

```diff
@@ -34,6 +38,7 @@
     <PackageReference Include="xunit" Version="$(XunitPackageVersion)" />
     <PackageReference Include="xunit.analyzers" Version="$(XunitAnalyzersPackageVersion)" />
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitRunnerVisualstudioPackageVersion)" />
+    <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="2.2.2" />
```

Comment:
I know this is test-only but have we used @natemcmaster's private packages elsewhere?

Reply:
I mimicked here what we did in the internal repo.